libldbrest/endpoints.go

```diff
@@ -104,8 +106,8 @@ func getItems(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
 		}
 	}
 
-	w.Header().Set("Content-Type", "application/json")
-	json.NewEncoder(w).Encode(multiResponse{nil, results})
+	w.Header().Set("Content-Type", "application/msgpack")
```

Comment:
yeah I just made the change in that chore branch of the client. I saw that as well, it's why I went with `application/msgpack`